pkg/diagnose/connectivity/core_endpoint.go

```diff
@@ -51,6 +60,26 @@ func diagnose(diagCfg diagnosis.Config, _ sender.DiagnoseSenderManager) []diagno
 	domainResolvers := resolver.NewSingleDomainResolvers(keysPerDomain)
 	client := forwarder.NewHTTPClient(config.Datadog)
 
+	// Logs diagnosing
+	endpoints, err := getLogsHTTPEndpoints()
+	url, err := logshttp.CheckConnectivityDiagnose(endpoints.Main)
+
+	name := fmt.Sprintf("Connectivity to %s", url)
+	diag := diagnosis.Diagnosis{
+		Name: name,
+	}
+	if err == nil {
```

Comment:
Maybe it would make sense to factor out lines `99-106` to a helper function and call it both here and and below? That way we get the same output format, and don't have to duplicate this code for the logs agent diagnostics.